src/main/scala/fs2/Process1.scala

```diff
@@ -94,6 +94,17 @@ object process1 {
           }
       }
       
+  /** Zip the elements of the input `Handle `with its indices, and return the new `Handle` */    
+  def zipWithIndex[F[_], I](implicit F: NotNothing[F]): Handle[F, I] => Pull[F, (I, Int), Handle[F, I]] = {
+    def go(n: Int): Handle[F, I] => Pull[F, (I, Int), Handle[F, I]] = {
+      Pull.receive({ 
```

Comment:
No need for the parents here, the braces are sufficient. Pull.receive { case chunk #: h => ... } all on one line.